src/pymor/models/transfer_function.py

```diff
@@ -24,10 +24,11 @@ class TransferFunction(CacheableObject, ParametricObject):
     dim_output
         The number of outputs.
     tf
-        The transfer function defined at least on the open right complex half-plane.
-        `tf(s, mu)` is a |NumPy array| of shape `(p, m)`.
+        The transfer function, given by a callable that takes a complex value `s` and,
+        if parametric, a |parameter value| `mu`.
+        The result of `tf(s, mu)` is a |NumPy array| of shape `(dim_output, dim_input)`.
     dtf
-        The complex derivative of `H` with respect to `s` (optional).
+        The complex derivative of `dtf` with respect to `s` (optional).
```

Comment:
```suggestion
        The complex derivative of `H` with respect to `s` (optional).
```
or something like "complex derivative of `tf`".